tests/func/test_data_cloud.py

```diff
@@ -39,7 +39,10 @@
 TEST_CONFIG = {
     Config.SECTION_CACHE: {},
     Config.SECTION_CORE: {Config.SECTION_CORE_REMOTE: TEST_REMOTE},
-    TEST_SECTION: {Config.SECTION_REMOTE_URL: ""},
+    TEST_SECTION: {
+        Config.SECTION_REMOTE_URL: "",
+        Config.SECTION_REMOTE_NO_TRAVERSE: False,
```

Comment:
```suggestion
        Config.SECTION_REMOTE_NO_TRAVERSE: True,
```

Reply:
This is the reason why it was set to `False`
https://github.com/iterative/dvc/pull/2411#discussion_r315355621

Reply:
@mroutis Got it. If it is only about SSHMocked test, then only modify it's _setup_cloud to set no_traverse to false. See TestRemoteGS._setup_cloud as an example. And please create a ticket for this, you've discovered a bug in our mockssh fixture.

Reply:
okok, will do, @efiop :+1:

Reply:
@efiop , created: https://github.com/iterative/dvc/issues/2412